search/search_integration_tests/pre_ranker_test.cpp

```diff
@@ -31,12 +31,13 @@
 #include "base/math.hpp"
 #include "base/stl_helpers.hpp"
 
-#include "std/algorithm.hpp"
-#include "std/iterator.hpp"
-#include "std/vector.hpp"
+#include <algorithm>
+#include <iterator>
+#include <vector>
```

Comment:
include utility for move

Reply:
https://en.cppreference.com/w/cpp/algorithm/move